### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- upgrade to Collabora 21.11.0.6

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: README.md [hunk 1]
```diff
@@ -33,7 +44,7 @@ LibreOffice-based online office suite with collaborative editing
 The settings of Collabora are at `https://office.domain.tld/loleaflet/dist/admin/admin.html`.
```
Comment (Taekiro) [inline README.md:44]: The new URI for the admin is : https://office.domain.tld/browser/dist/admin/admin.html

File: README_fr.md [hunk 2]
```diff
@@ -29,7 +39,7 @@ Suite office en ligne et collaborative, basée sur LibreOffice
 Les paramètres de Collabora sont à `https://office.domain.tld/loleaflet/dist/admin/admin.html`.
```
Comment (Taekiro) [inline README_fr.md:39]: The new URI for the admin is : https://office.domain.tld/browser/dist/admin/admin.html

File: manifest.json [hunk 3]
```diff
@@ -6,13 +6,12 @@
         "en": "LibreOffice-based online office suite with collaborative editing",
         "fr": "Suite office en ligne et collaborative, basée sur LibreOffice"
     },
-    "version": "6.4.10~ynh2",
+    "version": "21.11.0.6~ynh1",
```
Comment (Taekiro) [inline manifest.json:9]: As I said in the original PR, I don't know what will happen when the package _coolwsd_ will be updated on the collabora deb, but I feel like a simple `apt upgrade` will upgrade the app (but perhaps the package is marked "on hold" by Ynh). I don't know how Yunohost apps deal with this kind of situation.

Comment (ericgaspar) [reply manifest.json:9]: you maybe right, at least user will be informed from the readme of the version being installed.

**Comment by Taekiro** [comment]
There is also a problem line 48 in the README_fr : `:warning:` (instead of avertissement/attention)

**Comment by ericgaspar** [comment]
@Taekiro Are you able to connect to `/browser/dist/admin/admin.html` path with Firefox?
I am able to connect with Chromium but not with Firefox. I have this problem with other apps as well 🤔

**Comment by Taekiro** [comment]
> @Taekiro Are you able to connect to `/browser/dist/admin/admin.html` path with Firefox? I am able to connect with Chromium but not with Firefox. I have this problem with other apps as well 🤔

No problem with Firefox (well the auto-completion doesn't work properly but it maybe due to some add-on that I use for my passwds).

(You missed a `:Attention:` => `:warning:` at the end of the line 48.)

**Comment by Thovi98** [comment]
> Are you able to connect to /browser/dist/admin/admin.html path with Firefox?

I can also connect with Firefox ! 

Thanks both for all your work here :D

File: manifest.json (hunk 3 above)
Comment (tituspijean) [reply manifest.json:9]: @Taekiro indeed the virtual packages generated by YunoHost mark the dependencies as "on hold".

Comment (Taekiro) [reply manifest.json:9]: OK, I didn't knew that. How to upgrade the app then? Does a simple change of the version in the manifest trigger the apt upgrade on the held package?

**Pull request merged**